npc/other/Global_Functions.txt

```diff
@@ -659,3 +660,97 @@ function	script	F_getpositionname	{
 			return "Unknown";
 	}
 }
+
+//////////////////////////////////////////////////////////////////////////////////
+// Platinum Skills Global Function: F_GetPlatinumSkills [mazvi]
+//////////////////////////////////////////////////////////////////////////////////
+function	script	F_GetPlatinumSkills	{
+	skill "NV_FIRSTAID",1,SKILL_PERM;
```

Comment:
It would be wise to add a safe check for summoner class, which shouldn't get first aid skill.